smoke-tests/spec/external_dns_spec.rb

```diff
@@ -1,39 +1,44 @@
 require "spec_helper"
 
 describe "external DNS" do
-  # let!(zone) { nil }
+  namespace = "child-#{readable_timestamp}"
+  let(:domain) { "child.parent.service.justice.gov.uk" }
+  let(:ingress_domain) { domain }
```

Comment:
I think these statements belong in the "when zone matches ingress domain" context.

Ideally, the thing that makes the context different from other contexts should be done at the top of that context, to make it crystal clear to the reader what you're testing.